src/lib/process/index.ts

```diff
@@ -0,0 +1,98 @@
+import { createNodeApi } from '../utils/polkadot.js'
+import { PROCESS_ID_LENGTH } from '../types/restrictions.js'
+import { defaultOptions, Options } from '../types/options.js'
+import { createProcessTransaction, disableProcessTransaction, getVersion } from './api.js'
+import { utf8ToHex } from './hex.js'
+import { Polkadot } from '../types/substrate.js'
+import { mapRestrictions } from './map.js'
+import { Restrictions } from '../types/restrictions.js'
+
+export type ProcessResult = {
```

Comment:
would be nice to have all type ins types/ folder as there are already few

Reply:
Also if impoirting a type it's good to import as type e.g.
```ts
import type { MyType } = from '../a.d.ts'
```

Reply:
Why restrictions is save as .js?

Reply:
the imports break when running tests if I remove `.js`. The actual file is `.ts`

Reply:
don't have an extension

Reply:
if I remove `.js` the import fails when tests are run https://nodejs.org/api/esm.html#esm_import_specifiers